src/components/CurriculumComponents/OakComponentsKitchen/SimpleRadio/index.tsx

```diff
@@ -112,6 +116,7 @@ export function RadioButton(
         alignItems: "center",
         marginBottom: "12px",
         opacity: props.disabled ? 0.4 : 1,
+        ...(props.isNoneHighlighted ? { paddingLeft: "12.8px" } : {}),
```

Comment:
Although this is being removed, we don't generally want to use `px` oak uses `rem`s for old shared components and sizing tokens for new components. The `marginBottom: "12px"` a few lines above is an error.